src/asset/asset-list-loader.js

```diff
@@ -0,0 +1,195 @@
+import { EventHandler } from '../core/event-handler.js';
+
+import { Asset } from './asset.js';
+
+/**
+ * Used to load a group of assets and fires a callback when all assets are loaded.
```

Comment:
It'd be fantastic to have an example here in JSDocs format on how to use it .. including how to retry when some assets fail to load